kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -47,53 +47,62 @@
 <script>
 
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-  import useUser from 'kolibri.coreVue.composables.useUser';
   import { mapGetters } from 'vuex';
+  import { LearnerDeviceStatus } from 'kolibri.coreVue.vuex.constants';
+  import useUserSyncStatus from '../composables/useUserSyncStatus';
+  import plugin_data from 'plugin_data';
 
   export default {
     name: 'StorageNotification',
     components: {},
     mixins: [commonCoreStrings],
     setup() {
-      const { isLearnerOnlyImport } = useUser();
```

Comment:
We should continue to use this rather than the isSubsetOfUsersDevice below. This will avoid weird interactions after one user has migrated to a learner only device, while others have not.